src/current/v23.1/changefeed-for.md

```diff
@@ -82,6 +82,8 @@ Option | Value | Description
 
 #### Avro limitations
 
+Creating a changefeed using Avro is available in Core and [{{ site.data.products.enterprise }} changefeeds](create-changefeed.html) with the [`confluent_schema_registry`](create-changefeed.html#confluent-registry) option.
```

Comment:
Do we still make mention of Core? Or are we only calling it Self-Hosted now and referring it to `CockroachDB {{ site.data.products.core }}` ?

Reply:
Ah, so this is not referring to Core as in Self-Hosted CockroachDB, it is actually [Core changefeeds](https://www.cockroachlabs.com/docs/v23.1/changefeed-for), which are our implementation of changefeeds that customers do not need an Enterprise license for. Thanks for checking that though!